ROADMAP.md

```diff
@@ -0,0 +1,95 @@
+# IPLD Roadmap (2023)
+
+**Status:** Draft
+
+This document is a *work in progress* while we:
+
+1. Attempt to gather stakeholder feedback on priorities
+2. Plan and understand resourcing to execute on the goals
+
+## Scope
+
+"IPLD" is referred to here in a broad sense as "the data layer of the distributed web" with an IPFS flavour. As such, IPLD tools, libraries and documents will tend to use the CID and the Multiformats suite for expressing and representing linking. IPLD aims to be able to represent a broad range of **content addressed** data, but its anchoring in the IPFS ecosystem presents some limitations in the scope of what is possible to encompass.
+
+The exact boundaries of IPLD are not strictly defined. Historically, data storage and data transfer have represented boundaries for IPLD. These are not strict boundaries but provide guidance on the areas of concern and what the "data layer" means. Most commonly, IPLD relates to tooling for serialising / deserialising binary data, representing that data in a consistent and coherent **linked data model** and providing tools and techniques for addressing, traversing and manipulating that data.
+
+## Focus Areas
+
+### Documentation
+
+1. Improved onboarding documentation and resources.
+2. Improved UX for consolidated docs/specs site (ipld.io)
+
+### Distributed Data Structures
+
+Specifications and documentation related to data structures to represent common application data over content addressed data. Including: filesystem data, ordered and unordered maps, sparse and dense arrays, encrypted and/or signed data.
```

Comment:
How is this different from Documentation above? Maybe worth renaming the "Documentation" section title to something like Onboarding and Usability?